yt_dlp/extractor/owncloud.py

```diff
@@ -0,0 +1,88 @@
+import urllib.parse
+
+from yt_dlp.extractor.common import InfoExtractor
+from yt_dlp.utils import (
+    determine_ext,
+    ExtractorError,
+    url_or_none,
+    urlencode_postdata,
+)
+
+
+class OwnCloudIE(InfoExtractor):
+    _INSTANCES_RE = r'''(?:
+                            sciebo\.de|
+                            [^\.]+\.sciebo\.de|
+                            cloud\.uni-koblenz-landau\.de|
+                        )'''
+    _VALID_URL = rf'''(?x)
+        (?P<server>https?://{_INSTANCES_RE})/s/
+        (?P<id>[\w\-\.]+)'''
+
+    _TESTS = [
+        {
+            'url': 'https://ruhr-uni-bochum.sciebo.de/s/wWhqZzh9jTumVFN',
+            'info_dict': {
+                'id': 'wWhqZzh9jTumVFN',
+                'ext': 'mp4',
+                'title': 'CmvpJST.mp4',
+            },
+        },
+    ]
+
+    def _real_extract(self, url):
+        server, video_id = self._match_valid_url(url).group('server', 'id')
+
+        webpage, urlh = self._download_webpage_handle(f'{server}/s/{video_id}', video_id, 'Downloading webpage')
```

Comment:
* Why `f'{server}/s/{video_id}'` instead of just `url`?
* Why use `urlh.geturl()`?

If just `url` is insufficient in either case, add test for it